timvt/layer.py

```diff
@@ -208,6 +226,12 @@ async def get_tile(
         **kwargs: Any,
     ):
         """Get Tile Data."""
+        # We only support TMS with valid EPSG code
+        if not tms.crs.to_epsg():
+            raise MissingEPSGCode(
+                f"{tms.identifier}'s CRS does not have a valid EPSG code."
+            )
```

Comment:
because we state that function have to accept `epsg` as input, we must raise an exccption if there is no epsg code!